content/soarian/dstu2/medications/medication-statement.md

```diff
@@ -17,32 +17,32 @@ To avoid patient confusion about the definitive active medication list, no in-pr
 
 ## Terminology Bindings
 
-<%= terminology_table(:soarian_medication_statement, :dstu2) %>
+### Contained Medication Bindings
+
+<%= terminology_table(:soarian_contained_medication, :dstu2) %>
 
 ## Custom Extensions
 
 The [bundle extension] can be returned, see possible codes in the [errors section] below.
 
   Status Extension:
 
-The URL for this status extension is defined as `Additional status`
+The URL for this status extension is defined as `http://cerner.hs.fhir.com/StructureDefinition/additional-status`
 
  ID                         | Value\[x] Type | Description
 ----------------------------|----------------|---------------------------------------------------------------------------------------------------------------------------------------------
-`MedicationStatement/Staus` | [`string`]     | This element describes the additional status information of the MedicationStatement. Possible values are 'New', 'Continued' and 'Modified'.
+`MedicationStatement/Staus` | [`string`]     | This element describes the additional status information of the MedicationStatement. Additional statuses has beed added to properly portray the status of the medications on the DMR as the existing statuses do not reflect the correct status of the medication.Possible values are `new`, `continued` and `modified`.
```

Comment:
what's the DMR? Discharge medication reconciliation list?
Maybe (the first sentence isn't adding anything):

This element describes the status of the medications on the last discharge medication reconciliation list, rather than the order status. Possible values are `new`, `continued` and `modified`.